Sass/style.css

```diff
@@ -0,0 +1,555 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: white;
+  font-family: "Lato", sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+}
+
+a {
+  text-decoration-line: none;
+}
+
+a:focus {
+  background-color: rgb(21, 19, 22);
+}
+
+.nav-bar {
+  background-color: rgb(246, 238, 238);
+  box-shadow: 0 2px 8px rgba(10, 9, 9, 0.26);
+  position: sticky;
+  display: block;
+  justify-content: space-between;
+  width: 100%;
+  min-height: 60px;
+}
+
+#nav-menu-cc {
+  border: 5px solid #ec5242;
+  padding: 8px;
+  width: 25%;
+  color: #ec5242;
+}
+
+.nav-bar .logo {
+  padding-left: 10px;
+  color: black;
+  font-weight: bolder;
+  font-size: 40px;
+}
+
+.nav-bar ul {
+  display: none;
+}
+
+.nav-bar button {
+  width: -moz-fit-content;
+  width: fit-content;
+  float: right;
+  border: none;
+  margin: 5px 10px 0 0;
+  background-color: transparent;
+}
+
+.courses img {
+  border-radius: 10px;
+  height: 60px;
+  width: 60px;
+  margin-bottom: 10px;
+}
+
+.nav-bar button img {
+  width: 50px;
+  height: 50px;
+}
+
+.home-page {
+  display: flex;
+  flex-direction: column;
+  min-height: 600px;
+  background-image: url(../images/main-background-img.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.home-page h1 {
+  color: #ec5242;
+  font-size: 50px;
+  font-family: "COCOGOOSE", sans-serif;
+  font-weight: 400;
+  text-transform: uppercase;
+}
+
+.home-page h1 strong {
+  color: #ec5242;
+  font-size: 40px;
+}
+
+.home-page h1 span {
+  color: #ec5242;
+  font-size: 20px;
+}
+
+.home-page .home-buttons-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+.home-page .home-buttons-div button {
+  margin-top: 15px;
+  width: 90%;
+  min-height: 60px;
+  text-align: center;
+  font-size: 20px;
+  color: rgb(255, 255, 255);
+  background-color: rgb(255, 0, 0);
+  border-radius: 25px;
+  margin-bottom: 20px;
+}
+
+.copy {
+  margin-top: 140px;
+  float: left;
+  font-weight: 700;
+}
+
+.copy b {
+  font-size: 30px;
+}
+
+.about-us {
+  display: flex;
+  flex-direction: column;
+  padding-top: 30px;
+  align-items: center;
+  background-image: url(../images/abt-bg.jpg);
+  width: 100%;
+}
```

Comment:
Great job @degisew 🤝

- [ ] Just like the previous reviewer pointed out. It is important that you follow the project requirements as much as possible. I can see your design is missing the images on the about section.
- Given design
![images-about](https://user-images.githubusercontent.com/63917725/183266787-d80c54f5-3287-4735-a02d-9f8625f17e24.PNG)

- Your design
![inages-your about](https://user-images.githubusercontent.com/63917725/183266798-f95ab012-192b-4ee4-9187-b5ca9d428fa8.PNG)